cookbook/common/leaf.mk

```diff
@@ -138,25 +138,10 @@ serialize: clean _pb_output docker_build
 
 
 .PHONY: register
-register: clean _pb_output docker_push
+register: docker_push
```

Comment:
I believe the old `make register` also did serialize... so maybe we change this to:
```suggestion
register: serialize docker_push
```